packages/@angular/cli/commands/help.ts

```diff
@@ -56,7 +56,7 @@ const HelpCommand = Command.extend({
 
       if (rawArgs.length > 0) {
         if (cmd === rawArgs[0]) {
-          this.ui.writeLine(command.printDetailedHelp(commandOptions));
+          this.ui.writeLine(command.printBasicHelp(commandOptions));
```

Comment:
But we are not checking whether detailed help is there or not as both the functions would exist on the command itself. One way to do is to remove the `printDetailedHelp` from Command prototype and above add this check. 

```